### DISCUSSION TITLE
Use of lead-acid battery models

### DISCUSSION THREAD

**Discussion by simdieudoboinzemwende** [pybamm-team/PyBaMM#4347] [General]
Hello everyone, first of all, I apologize for being a bit lengthy. I primarily work on lithium batteries, but this time, I want to conduct an experiment to simulate certain aging mechanisms of a lead-acid battery. My usual approach involves a long experiment with charge and discharge cycles. However, this time, I’m encountering errors. Apparently, with lead-acid battery models, we cannot use cutoff voltages as conditions to define the cycles. I’m truly stuck at this point because if I use time as the end-of-cycle condition, it fails over time due to overlaps. 
Another thing I’ve noticed is that with this model (lead_acid.Full), if we set the number of cells in series to 6, the output voltage remains the same as that of a single cell. 
My final concern is how to simulate calendar aging. Here, I’m trying to understand the impact of ambient storage temperature on battery aging

**Comment by valentinsulzer** [comment]
Lead-acid should work with cut-off voltages. Please share your script. The issue might be that you need to use the 10.5V / 12V limits, not the ~2V limits.
The output that varies with number of cells in series is "Battery voltage [V]".
To simulate calendar aging you could try doing `Full({"hydrolysis": "true"})`.

**Comment by simdieudoboinzemwende** [comment]
Okay, thank you! I’ll give it a try. Thanks again for the response!
